main/infrastructure-tests/config/settings/example.yml

```diff
@@ -0,0 +1,27 @@
+# The AWS region where the service workbench application is deployed
+awsRegion: us-east-1
+
+# AWS profile to use for deployment. You must provide this value if isBuildServer = false
+awsProfile:
```

Comment:
I'd fill out an example value for attributes here.

Reply:
I'll add an example value in the comments, since I don't want people to accidentally deploy to `default`. I want to make sure that they purposefully pick a profile.